hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -345,8 +349,8 @@ def write(self, vals):
     @api.multi
     def name_get(self):
         # week number according to ISO 8601 Calendar
-        return [(r['id'], _('Week ') + str(
-            r['date_start'].isocalendar()[1]))
+        return [(r['id'], _('Week %s') % (str(
+            r['date_start'].isocalendar()[1])))
```

Comment:
@pedrobaeza is there a way that a comma is needed to force it being a tuple?

Reply:
I don't get the problem

Reply:
https://github.com/OCA/hr-timesheet/pull/183#issuecomment-476579082

Reply:
@alexey-pelykh you should put the comma or remove some parenthesis, ie, `'%s' % value` or `'%s' % (value, )` but not '`%s' % (value)`.

Reply:
https://github.com/OCA/hr-timesheet/pull/222